libkarel.py

```diff
@@ -68,6 +72,13 @@ def __init__(self, string):
         self.__zumbadores = {(x['x'], x['y']): x['zumbadores']
                              for x in lista_zumbadores}
 
+        limites = [
+            {
+                'comando': x.attrib['nombre'],
+                'max': int(x.attrib['maximoNumeroDeEjecuciones']),
+            } for x in self.root.findall('condiciones/comando')]
+        self.__limites = {x['comando']: x['max'] for x in limites}
```

Comment:
esto lo puedes hacer en un solo paso:

```suggestion
        self.__limites = {
            x.attrib['nombre']: int(x.attrib['maximoNumeroDeEjecuciones'])
            for x in self.root.findall('condiciones/comando')
        }
```